Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine AS build
+# packages needed for node-gyp and dupe-krill
+RUN apk add --update --no-cache python make g++ rust cargo
+# use a non-root user to make commands such as npm post-install scripts safer
+USER node
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin:/home/node/.cargo/bin
+# RUN cargo install dupe-krill
+# copy by default will make the root user own the files, we want the node user to own the files
+COPY --chown=node:node package*.json .snyk /home/node/app/
+WORKDIR /home/node/app
+RUN npm install
+COPY --chown=node:node . /home/node/app/
+RUN npm run build
+RUN npm prune --prod
```

Comment:
There is a `FROM` below, so it _probably_ doesn't matter, but you may consider combining these two `RUN` statements.

See https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better

Reply:
Thanks, that's a great piece of information, I'll do it :-)